common/submissions.py

```diff
@@ -59,29 +57,33 @@ def checkout_to_team_master(
     for tag_ref in repo.tags:
         repo.delete_tag(tag_ref)
 
-    repo_name = f"{hw_name}-{team}"
-    repo.create_remote(team, f"git@github.com:{HW_ORG}/{repo_name}.git")
+    repo.create_remote(team, f"git@github.com:{team_repo_id}.git")
 
     repo.git.fetch(team, "--tags")
     repo.git.fetch(team, "master")
 
-    # Let's checkout to the team's master branch to start out.
+    # Let's checkout to the team's branch.
+    team_branch = f"{team}-{branch_name}"
     try:
-        repo.git.branch('-D', team)  # Just in case it exists already.
+        repo.git.branch('-D', team_branch)  # Just in case it exists already.
     except git.GitError:
         # This branch doesn't exist.
         pass
 
-    repo.git.checkout("-b", team, f"{team}/master")
+    repo.git.checkout("-b", team_branch, f"{team}/{branch_name}")
 
     return True
 
+def checkout_to_team_master(
```

Comment:
yeah but i left it because cs4118 only ever checks out to master, so its j kinda nicer looking. now that i think about it tho, it might make more sense to do `branch_name = "master"` default param instead, so then cs4118 would j replace every call to `checkout_to_team_master` with `checkout_to_team_branch`, i think i'll do that